src/components/ens-profile/ActionButtons/MoreButton.tsx

```diff
@@ -117,12 +117,12 @@ export default function MoreButton({
       if (actionKey === ACTIONS.OPEN_WALLET) {
         if (!isSelectedWallet) {
           setIsSearchModeEnabled?.(false);
-          switchToWalletWithAddress(address);
+          switchToWalletWithAddress(address!);
```

Comment:
The whole component has `address` as optional, so it's not clear to me that `address` will be specified here. It should be, based on how and why we add this option to the context menu, but this relation is very implicit and can easily be broken.

Maybe we can add `&& !!address` to the `if` above? IMO better to be fully safe rather than just satisfy type system.